README.rst

```diff
@@ -117,15 +119,6 @@ There are two steps that should be done when adding a new plugin to InfraRed:
     For more details on how to use this module, please visit the 'clg' module `homepage <http://clg.readthedocs
     .org/en/latest/>`_.
 
-2. Creating a default spec file (default.ini). 
-    This file should contain the default values for the command line arguments. All the default values should go under the name section names as a new plugin. Example::
-      
-      [virsh]
-      topology=all-in-one.yml
-      network=default.yml
-      ssh-key=~/.ssh/id_rsa
-      ssh-user=root
-
 3. Creating settings files.
```

Comment:
This one should be updated to be the second step now (and not the third).
RST supports '#' in enumerated lists for auto-enumeration. Please us it.